packages/manager/modules/pci/src/projects/project/instances/instance/instance.html

```diff
@@ -88,6 +89,7 @@
                 <oui-tile-button
                     data-on-click="$ctrl.endRescueInstance()"
                     data-ng-if="$ctrl.instance.status === 'RESCUE'"
+                    data-disabled="$ctrl.instance.isShelvingOrUnshelving()"
```

Comment:
Don't get it, because if condition is `$ctrl.instance.status === 'RESCUE'` but disabled condition is `this.instance.status === INSTANCE_STATUS.SHELVING || this.instance.status === INSTANCE_STATUS.UNSHELVING`

The conditions will never be ok.

Reply:
Thanks for this observation.
Right the both i and disabled are no compliance.
I removed the disabled one.